cmd/utils.go

```diff
@@ -1045,3 +1045,23 @@ func checkTime() {
 		}
 	}
 }
+
+func IsEmptyDir(name string) bool {
+	f, err := os.Open(name)
+
+	if err != nil {
+		return true
+	}
+	defer f.Close()
+
+	_, err = f.Readdirnames(1)
+	if err == io.EOF {
+		return true
+	}
+	return false
+}
+
+func fileDoesNotExist(filename string) bool {
```

Comment:
I just realized utils.go already has a function like this called `exists()`. We should probably use that function, it has a little better error handling. As this one is now, the error handling isn't quite right. Let's discuss.